hydro-generation/preprocess-data.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Preprocess IRENA hydro generation data\n",
     "\n",
-    "This notebook preprocesses IRENA data from their \"Renewable energy statistics\" publication. It reduces the dataset to generation of hydro power in the relevant countries. This notebook creates the `./data/irena/hydro-generation-europe.csv` dataset. The original IRENA dataset is not part of this repository and must be retrieved manually to run this notebook."
+    "This notebook preprocesses IRENA data from their \"Renewable energy statistics\" publication. It reduces the dataset to generation of hydro power in the relevant countries. This notebook creates the `./hydro-generation/hydro-generation.csv` dataset. The original IRENA dataset is not part of this repository and must be retrieved manually to run this notebook."
```

Comment:
Update to remove 'It reduces the dataset to generation of hydro power in the relevant countries' to match the removal of `-europe`